CHANGES.rst

```diff
@@ -15,6 +15,7 @@ New Builtins
 #. ``Accuracy``
 #. ``ClebschGordan``
 #. ``Curl`` (2-D and 3-D vector forms only)
+#. ``DisplayForm``
```

Comment:
Also there is that MANIFEST file you added that needs updating.

Reply:
I expected that the CI complains about this...

Reply:
We should see why it doesn't.